src/main/java/de/teamlapen/vampirism_integrations/mca/EntityAngryVillagerMCA.java

```diff
@@ -29,64 +39,59 @@
  */
 public class EntityAngryVillagerMCA extends EntityVillagerVampirismMCA implements IAggressiveVillager {
 
-    private ItemStack pitchforkStack;
-    private AxisAlignedBB area;
-
-
-    public EntityAngryVillagerMCA(World world) {
-        super(world);
-        pitchforkStack = new ItemStack(MCACompatREFERENCE.pitchfork);
```

Comment:
Why shouldn't they have pitchforks? Are they unable to use items or was it not finished yet?